### ISSUE TITLE
Error being generated when using Zipkin

### ISSUE THREAD

**Issue by trajano** [containous/traefik#5646]
### Do you want to request a *feature* or report a *bug*?

Bug

`level=error msg="opentracing: Invalid Inject/Extract carrier"` appears in the logs when using zipkin

https://community.containo.us/t/level-error-msg-opentracing-invalid-inject-extract-carrier-when-using-zipkin/1461

### What did you do?

```
      - --tracing.zipkin.httpEndpoint=http://zipkin:9411/api/v2/spans
```

### What did you expect to see?

No errors

### What did you see instead?

Errors in the log

`level=error msg="opentracing: Invalid Inject/Extract carrier"` 

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.0.2
Codename:     montdor
Go version:   go1.13.1
Built:        2019-10-09T19:26:05Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

I don't think it's relevant.

```toml
[tls.options]

  [tls.options.default]
    minVersion = "VersionTLS12"

    cipherSuites = [
        "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
        "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
        "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
        "TLS_AES_128_GCM_SHA256",
        "TLS_AES_256_GCM_SHA384",
        "TLS_CHACHA20_POLY1305_SHA256"
    ]

    sniStrict = true

  [tls.options.intranet]
    minVersion = "VersionTLS12"

    cipherSuites = [
        "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
        "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
        "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
        "TLS_AES_128_GCM_SHA256",
        "TLS_AES_256_GCM_SHA384",
        "TLS_CHACHA20_POLY1305_SHA256"
    ]

    sniStrict = true
    [tls.options.intranet.clientAuth]
      caFiles = ["/ca.pem"]
      clientAuthType = "RequireAndVerifyClientCert"

[http.middlewares]
  [http.middlewares.compress-all.compress]

  [http.middlewares.https-only.redirectScheme]
    scheme = "https"
    permanent = true

  [http.middlewares.security-headers.headers]
    BrowserXssFilter = true
    ContentTypeNosniff = true
    ForceSTSHeader = true
    FrameDeny = true
    SSLRedirect = true
    STSIncludeSubdomains = true
    STSPreload = true
    STSSeconds = 315360000

  [http.middlewares.strip-prefix.chain]
    middlewares = [ "strip-prefix-1", "strip-prefix-2"]

  [http.middlewares.strip-prefix-1.redirectregex]
    regex="^(https?://[^/]+/[a-z0-9_]+)$"
    replacement="${1}/"
    permanent = true

  [http.middlewares.strip-prefix-2.stripprefixregex]
    regex = [ "/[a-z0-9_]+" ]

```

```
      - --global.checknewversion=false
      - --global.sendanonymoususage=false

      - --accesslog
      - --api
      - --api.insecure
      - --ping

      - --certificatesresolvers.default.acme.httpchallenge.entrypoint=http
      - --certificatesresolvers.default.acme.email=support@trajano.net
      - --certificatesresolvers.default.acme.storage=/etc/traefik/acme/default.json

      - --entrypoints.http.address=:80
      - --entrypoints.https.address=:443

      - --providers.docker.endpoint=tcp://daemon:2375
      - --providers.docker.exposedByDefault=false
      - --providers.docker.swarmMode=true
      - "--providers.docker.constraints=!Label(`intranet`, `true`)"
      - '--providers.docker.defaultrule=Host(`trajano.net`) && PathPrefix(`/{{ .Name | trimPrefix (index .Labels "com.docker.stack.namespace") | trimPrefix "_" | trimPrefix "-" }}`)'

      - --providers.file.filename=/traefik-conf.toml

      - --tracing.zipkin.httpEndpoint=http://zipkin:9411/api/v2/spans
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
(paste your output here)
```

**Comment by ddtmachado** [comment]
This seems related to a recent modification on zipkin-go-opentracing that introduced a bug.

I just opened an [issue](https://github.com/openzipkin-contrib/zipkin-go-opentracing/issues/151) and [PR]( https://github.com/openzipkin-contrib/zipkin-go-opentracing/pull/152) with a fix.

**Comment by jcchavezs** [comment]
Thanks for raising this issue. We just released v0.4.4 and I opened a PR for fixing this issue: https://github.com/containous/traefik/pull/5754

**Comment by traefiker** [comment]
Closed by #5754.
